packages/frontend/src/pages/formatLargeNumber.ts

```diff
@@ -2,13 +2,16 @@ const units = ['', 'K', 'M', 'B', 'T']
 
 const HAIR_SPACE = '\u200a'
 
-export function formatLargeNumber(value: number): string {
+export function formatLargeNumber(value: number | bigint): string {
```

Comment:
@sin6pi7 @sz-piotr what is the input like?

Reply:
Originally in L2BEAT the input is the USD TVL of something, e.g. 1234567890.1234 would be formatted as 1.23B

Because you do not multiply the bigint that you're passing you are actually formatting it incorrectly, so `1n` is going to be formatted as `0.01` instead of `1.00`